### PR TITLE
add ecobee ventilator documentation

### PR DESCRIPTION
**Pull request by marcolivierarsenault**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/83645
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ecobee.markdown [hunk 1]
```diff
@@ -128,6 +130,26 @@ The ecobee climate entity has some extra attributes to represent the state of th
 | `equipment_running` | This is a comma-separated list of equipment that is currently running.
 | `fan_min_on_time` | The minimum amount of time (in minutes) that the fan will run per hour. This is determined by the minimum fan runtime setting which can be changed in the ecobee app or on the thermostat itself.
 
+## Ventilator
+
+### Concepts
+
+The ecobee thermostat supports the addition of an accessory. If you have a air exchanger (ventilator, hrv or erv) you can control it via the Ventilator entity.
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]:
```suggestion
The ecobee thermostat supports the addition of an accessory. If you have an air exchanger (ventilator, HRV, or ERV), you can control it via the Fan entity.
```

File: source/_integrations/ecobee.markdown [hunk 2]
```diff
@@ -128,6 +130,26 @@ The ecobee climate entity has some extra attributes to represent the state of th
...
+
+The _state_ is the state of the ventilator. (on or off). It tells you if the ventilator is currently running.
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]:
```suggestion
The _state_ is the state of the fan. (on or off). It tells you if the ventilator is currently running.
```

Comment (frenck) [reply source/_integrations/ecobee.markdown]: Don't get me wrong, this feels a bit obvious. Should we drop this line?

File: source/_integrations/ecobee.markdown [hunk 3]
```diff
@@ -56,7 +58,7 @@ Your new application will now appear on the left. Upon clicking on the applicati
 1. In the **Settings** -> **Devices & Services** menu, click **+** and then select "ecobee" from the pop-up menu.
 2. In the pop-up box, enter the API key you obtained from ecobee's [developer portal](https://ecobee.com/developers).
 3. In the next pop-up box, you will be presented with a unique four-character PIN code which you will need to authorize in the [ecobee consumer portal](https://www.ecobee.com/consumerportal/index.html). You can do this by logging in, selecting **My Apps** from the hamburger menu, clicking **Add Application** on the left, entering the PIN code from Home Assistant, clicking **Validate** and then **Add Application** in the bottom right.
-4. After authorizing the app with ecobee, return to Home Assistant and click **Submit**. If the authorization was successful, a configuration entry will be created and your thermostats and sensors will be available in Home Assistant.
+4. After authorizing the app with ecobee, return to Home Assistant and click **Submit**. If the authorization was successful, a configuration entry will be created and your thermostats, ventilators and sensors will be available in Home Assistant.
```
Comment (frenck) [inline source/_integrations/ecobee.markdown:61]: We call them fans in Home Assistant.

```suggestion
4. After authorizing the app with ecobee, return to Home Assistant and click **Submit**. If the authorization was successful, a configuration entry will be created and your thermostats, fans, and sensors will be available in Home Assistant.
```

File: source/_integrations/ecobee.markdown [hunk 4]
```diff
@@ -128,6 +130,26 @@ The ecobee climate entity has some extra attributes to represent the state of th
...
+
+The turn-on turn-off control of the fan, is powering the 20 min manual mode (Same as on the ecobee device). So it is expected behavior that when you turn it on, it only stays on for 20 min.
+
+### Attributes
+
+The ecobee ventilator entity has some extra attributes to represent the state of the ventilator.
+
+| Name | Description |
+| ---- | ----------- |
+| `is_on` | Current status of the ventilator. The possible values are True or False.
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: `is_on` is not an attribute of an fan entity, but an property of the implementation of an fan entity. 

```suggestion
```

File: source/_integrations/ecobee.markdown [hunk 5]
```diff
@@ -128,6 +130,26 @@ The ecobee climate entity has some extra attributes to represent the state of th
...
+| `ventilator_type` | The type of ventilator present for the thermostat. The possible values are `none`, `ventilator`, `hrv`, and `erv`.
+| `ventilator_min_on_time_home` | The minimum amount of time (in minutes) that the ventilator will run per hour, when you are home. This is determined by the minimum ventilator runtime setting which can be changed in the ecobee app or on the thermostat itself.
```
Comment (frenck) [inline source/_integrations/ecobee.markdown:143]:
```suggestion
| `ventilator_min_on_time_home` | The minimum amount of time (in minutes) that the ventilator will run per hour when you are home. This is determined by the minimum ventilator runtime setting, which can be changed in the ecobee app or on the thermostat itself.
```

File: source/_integrations/ecobee.markdown [hunk 6]
```diff
@@ -128,6 +130,26 @@ The ecobee climate entity has some extra attributes to represent the state of th
...
+| `ventilator_min_on_time_away` | The minimum amount of time (in minutes) that the ventilator will run per hour, when you are away. This is determined by the minimum ventilator runtime setting which can be changed in the ecobee app or on the thermostat itself.
```
Comment (frenck) [inline source/_integrations/ecobee.markdown:144]:
```suggestion
| `ventilator_min_on_time_away` | The minimum amount of time (in minutes) that the ventilator will run per hour when you are away. This is determined by the minimum ventilator runtime setting, which can be changed in the ecobee app or on the thermostat itself.
```

File: source/_integrations/ecobee.markdown [hunk 7]
```diff
@@ -211,3 +235,21 @@ Enable/disable Smart Home/Away and Follow Me modes.
 | `entity_id`            | yes      | ecobee thermostat on which to set occupancy modes |
 | `auto_away`            | yes      | true or false                                     |
 | `follow_me`            | yes      | true or false                                     |
+
+### Service `ecobee.set_ventilator_min_on_time_home`
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: Should this be a `number` entity instead?

File: source/_integrations/ecobee.markdown [hunk 8]
```diff
@@ -211,3 +235,21 @@ Enable/disable Smart Home/Away and Follow Me modes.
...
+
+Set the minimum ventilator on time for home mode.
+
+| Service data attribute        | Optional | Description                                                                                                              |
+| ----------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `entity_id`                   | yes      | String or list of strings that point at `entity_id`'s of fan devices to control. Use `entity_id: all` to target all. |
+| `ventilator_min_on_time_home` | no       | integer (between 0 and 60, e.g. 30)                                                                                      |
+
+### Service `ecobee.set_ventilator_min_on_time_away`
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: Shouldn't this be a `number` entity instead?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ecobee.markdown (hunk 3 above)
Comment (marcolivierarsenault) [reply source/_integrations/ecobee.markdown:61]: @frenck Through the doc, I tried to separate fan from ventilator. 
I know Home assistant entity is a fan. But given the Ecobee device has:
- a Fan (for the furnace) which is included in the climate entity
- a Ventilator, (implemented from a fan entity) 

I think it makes the doc more clear to use ventilator. 

That being said, I have no strong feeling if you prefer to move to `fan`

File: source/_integrations/ecobee.markdown (hunk 7 above)
Comment (marcolivierarsenault) [reply source/_integrations/ecobee.markdown]: done

File: source/_integrations/ecobee.markdown (hunk 8 above)
Comment (marcolivierarsenault) [reply source/_integrations/ecobee.markdown]: done

**Comment by marcolivierarsenault** [comment]
@frenck applied your comments except the fan/ventilator comments.

Let me know what you think

**Review by frenck — APPROVED** [review]
Thanks, @marcolivierarsenault 👍 

../Frenck

**Pull request merged**